Library/OcDevicePathLib/OcDevicePathLib.c

```diff
@@ -1231,3 +1231,49 @@ OcGetNumDevicePathInstances (
 
   return NumInstances;
 }
+
+BOOLEAN
+OcDevicePathHasFilePathSuffix (
+  IN EFI_DEVICE_PATH_PROTOCOL  *DevicePath,
+  IN CHAR16                    *Suffix,
+  IN UINTN                     SuffixSize
```

Comment:
A bit unpleasant that can access one char paste the Suffix at least. Should document…